src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,39 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
```

Comment:
Try to resolve this issue
![image](https://user-images.githubusercontent.com/70009205/220151370-b71e5936-8016-44e6-b74e-daac2c77511b.png)

Reply:
Don't ignore this comment. Try to add `.jsx` 
BTW where is this file on git. Add it for review 
![image](https://user-images.githubusercontent.com/70009205/220179346-5d779ce9-9fc4-4332-ba8f-9eb4075f140b.png)

If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review